files/logstash-configs/22-redmine.conf

```diff
@@ -23,6 +23,9 @@ filter {
         match => {"message" => "%{REDMINE_SUCCESSFUL_LOGIN}"}
         add_tag => "successful_login"
       }
+      date {
+        match => ["redmine_auth_timestamp", "yyyy-MM-dd HH:mm:ss"]
```

Comment:
@conorsch ah, hang on one minute there. The timestamps on these log events are actually different formats. One has the offset and the other doesn't. So it works best to keep them separate.